Ardupilog.m

```diff
@@ -42,13 +43,42 @@
             % THE MAIN CALL: Begin reading specified log file
             readLog(obj);
         end
+        
+        function [] = countMsgs(obj)
+            if obj.fileID==-1
+                error('Log file should be open upon call');
+            end
+            
+            readsize = 1024*1024;
+                        
+            indices = [];
+            carry = [];
+                        
+            while feof(obj.fileID)==0
+                batch = [carry fread(obj.fileID,readsize)'];
+                if isempty(batch)
+                    break
+                end
+                if batch(end)==163
+                    carry = 163;
```

Comment:
If the 1nd byte of the start code lies at the end of the block, append it in front of the next block